chore: update WG files (#3497)
asyncapi/community: docs/020-governance-and-policies/WORKING_GROUPS.md

### Definition of a Working Group

A Working Group is a group of people in our community who share a common interest beyond GitHub repositories. These groups address cross-cutting concerns, ecosystem initiatives, and community goals that require ongoing and coordinated effort.

### Formation Requirements
To form a Working Group, all of the following prerequisites must be met:

> - At least 3 founding members who are active contributors to the AsyncAPI community, and approval from the TSC (Technical Steering Committee).

Reviewer: Approval from the TSC to agree on what? (Both old and new versions are a bit ambiguous on that)
- Creation of the working group?
- Founding members?
- Both?

Author: Thanks for pointing it out, added clarity

 - Work closely with the co-chair to monitor ongoing activities, proactively share updates, and coordinate responsibilities so there is always available leadership and seamless continuity.
 - Communicate all discussions, key updates, and group decisions through official AsyncAPI channels, ensuring the broader community remains informed and engaged.

Lastly, it is mandatory for all chairpersons to take the [Leading High-Performance Working Group Meetings](https://training.linuxfoundation.org/training/leading-high-performance-working-group-meetings-lfc120) course as this will help in leading effectively.

### Project Planning and Tracking

It is recommended that the Working Group use an associated project management tool, such as [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/creating-projects/creating-a-project) or another tool, to outline their roadmap. It's up to the Working Group to decide the project's structure. E.g., Kanban, Shape Up, TO-DO list, etc.

The group's name, members, goals, and other relevant information must be added to the `WORKING_GROUPS.yaml` file in the `community` repository.

### Decision Making

> A Working Group has no authority or power over the roadmap of the projects they may impact. It's up to the repositories' maintainers (code owners) to approve or reject the pull requests.  Therefore, it's advisable to either have maintainers of the impacted projects in the Working Group or to ensure everyone agrees on the roadmaps for the different repositories/projects.

Reviewer: ```suggestion
A Working Group does not have authority over the roadmaps of the projects it may influence. Approval or rejection of pull requests remains the responsibility of the repositories' maintainers (code owners). To ensure alignment, it is recommended to include maintainers of the affected projects in the Working Group or to establish consensus on the roadmaps for the relevant repositories and projects.
```

A Working Group must discuss ideas and solutions in public and communicate, through official channels, all relevant discussions and implementations it wishes to suggest to others.

### Meetings

> It is recommended that the Working Group schedule meetings using the methods described at [MEETINGS_ORGANIZATION.md](https://www.asyncapi.com/docs/community/060-meetings-and-communication).

Reviewer: ```suggestion
The Working Group should schedule meetings using methods described at [MEETINGS_ORGANIZATION.md](https://www.asyncapi.com/docs/community/060-meetings-and-communication).
```

Author: We're recommending not that they should do it because there are other maintainers who are doing it completely different for what we're suggesting.
